network/src/client.rs

```diff
@@ -13,7 +13,7 @@ use crate::{
 
 pub const LOG_TARGET: &str = "nexus-network::client";
 
-// const URL: &str = "http://35.209.216.211:80/api";
+// const URL: &str = "http://35.209.216.211:80/core";
```

Comment:
I don't think this file needs any changes. Probably find+replace mistake

Reply:
Yep that was, sorry, will fix